tests/vast-test.vala

```diff
@@ -237,8 +311,9 @@ int main (string[] args) {
         assert (29 == a.slice ({0, 0}, {-1, -1}).shape[1]);
 
         // reverse stride
-        assert (10 == a.slice ({10, 10}, {0, 0}).shape[1]);
-        assert (-1 * sizeof (int64) == a.slice ({10, 10}, {0, 0}).strides[1]);
+        // this shall die, slice cannot handle this because it assumes step == 1
+        // assert (10 == a.slice ({10, 10}, {0, 0}).shape[1]);
+        // assert (-1 * sizeof (int64) == a.slice ({10, 10}, {0, 0}).strides[1]);
```

Comment:
There is reversed slicing the the slice method of array doesn't support it -- it always does step == 1. 

In Python if you set from >= to without explicitly say step=-1 you get an empty array.

I think we need to follow that convention -- but because we insisted this slice function only describes `a:b:`, there is no way to pass in a step.